docs/api-docs/store-management/shipping/shipper-hq.md

```diff
@@ -17,46 +17,46 @@ These field values can be set in the Control Panel by the merchant as well as us
-| id | number | ID of the Metafield. **read-only** |
+| permission_set | string | This must be set to `write`. Setting to any other field will cause unintended behavior. (**required**) |
+| key | enum | Members: `shipping-groups`, `shipping-origins`, `shipping-dimensional-rules`. (**required**) |
+| value | JSON encoded string | This will be the name of the attribute type in ShipperHQ. Input the name exactly how it is listed within ShipperHQ. To pass in more than one Shipping Origin, Origin Location or Dimensional Rule, use an array with a comma-separated list. White spaces and special characters need to be escaped. Must be a JSON encoded string `"[\"Origin Name from SHQ-1\",\" Origin Name from SHQ-2\"]"`. (**required**) |
+| namespace | string | This will always be `shipping.shipperhq`. (**required**) |
+| resource_type | string | Either `product` or `variant`.(**read-only**)|
+| resource_id | string | ID of the `product` or `variant`. (**read-only**) |
+| description | string | ShipperHQ shipping origins associated with this product/sku. (**read-only**) |
+| date_created | string | Date and time of the metafield's creation. (**read-only**) |
+| date_modified | string | Date and time when the metafield was last updated. (**read-only**) |
+| id | number | ID of the Metafield. (**read-only**) |
 
 <div class="HubBlock--callout">
 <div class="CalloutBlock--warning">
 <div class="HubBlock-content">
     
 <!-- theme: warning -->
-### Entering Metafield Data
-> The metafields must be entered exactly, otherwise they will not work. Since metafields can also be used to store information against a product, if the <code>namespace</code> for example is set to shipping and not <code>shipping.shipperhq</code>, it will not update the shipping information.
+### Entering metafield data
+> You must enter the metafields exactly, otherwise, they will not work. Since metafields can store information against a product, if the <code>namespace</code> for example is set to shipping and not <code>shipping.shipperhq</code>, it will not update the shipping information.
 
 </div>
 </div>
 </div>
 
-## Control Panel Behavior
+## Control panel behavior
 
-When Shipping Origins are set using the API, the fields cannot be deleted using the Control Panel. A /DELETE request must be sent to the metafields resource.
+When you set Shipping Origins using the API, you cannot delete the fields using the Control Panel. You must send a /DELETE request to the metafields resource.
 
-## Add ShipperHQ Metafield
+## Add ShipperHQ metafield
 
 <!--
 title: "SHQ Add Metafield Request"
 subtitle: "/POST https://api.bigcommerce.com/stores/{{store_hash}}/v3/catalog/products/{{product_id}}/metafields"
 lineNumbers: true
 -->
 
-**Example Request Add ShipperHQ Metafields**  
+**Example request add ShipperHQ metafields**  
 `/POST https://api.bigcommerce.com/stores/{{store_hash}}/v3/catalog/products/{{product_id}}/metafields`

```

Comment:
I think we can remove this code since we are providing a link to the endpoint in Resources.

Reply:
Done